locales/base/translation.json

```diff
@@ -209,8 +209,8 @@
     "body": "You’ll get rewarded weekly on your Celo stablecoin with the highest value (cEUR, cUSD, cREAL).",
     "dismiss": "Dismiss"
   },
-  "superchargingTitle": "Your {{token}} is being Supercharged",
```

Comment:
Wait will removing token cause a crash on v1.53?

Reply:
Tested locally on 1.53 with the new strings and it appears to be fine to remove `{{token}}`.